source/input_conv.cpp

```diff
@@ -69,18 +69,6 @@ void Input_Conv::Convert(void)
 	GlobalV::PSEUDORCUT = INPUT.pseudo_rcut;
 	GlobalV::RENORMWITHMESH = INPUT.renormwithmesh;
 
-
-	// Electrical Field
-	GlobalV::EFIELD = INPUT.efield;
-	Efield::edir = INPUT.edir;
```

Comment:
How about the `Efield` class and `GlobalV::EFIELD`? Did them used in program?

Reply:
> How about the `Efield` class and `GlobalV::EFIELD`? Did them used in program?

Yes, but @mohanchen wants to delete these input parameters temporarily.

Reply:
> > How about the `Efield` class and `GlobalV::EFIELD`? Did them used in program?
> 
> Yes, but @mohanchen wants to delete these input parameters temporarily.

"Temporarily" means these features might add back, but we're not parsing input arguments? Sorry but I can not get the reason here.
@mohanchen Would you explain your concern in detail?

Reply:
I suggest to delete the Electric-Field parameters, we will refactor this function and redesign the input parameters in the near future (in the Operator module)

Reply:
> I suggest to delete the Electric-Field parameters, we will refactor this function and redesign the input parameters in the near future (in the Operator module)

@mohanchen Sorry, I'm still confused: why re-designing input parameter requires removing current parameters now?

Reply:
We want to clean the parameters, anyone seeing these parameters are wasting time. Could you ask me in person because we need to refactor 2.2 as soon as possible, I don't have time to explain to you all these details regarding the input parameters for so many days! You can of course ask me other questions regarding other projects on this github, but we need to push the 2.2 version as soon as possible because some other reasons that I don't have time to explain to EVERYONE either, and the refactor of input parameters, I also don't want to explain to EVERYONE! It is wasting too much time!

Reply:
@mohanchen OK, I see.